scripts/datasets/light_microscopy/check_mouse_embryo.py

```diff
@@ -1,20 +1,31 @@
-import os
-import sys
-
-from torch_em.data.datasets import get_mouse_embryo_loader
 from torch_em.util.debug import check_loader
+from torch_em.data.datasets import get_mouse_embryo_loader
```

Comment:
Can you please reset this file? There is a new logic to the scripts to set the ROOT directory globally in `../util.py`. Please use this logic to set your appropriate root dir rather than changing it in the scripts, as this will otherwise lead to many meaningless changes.

Reply:
I see, didn't spot it carefully. Thanks for mentioning. I'll make the other datasets from my side consistent as well.